projects/packages/search/src/dashboard/components/record-meter/lib/record-info.js

```diff
@@ -98,6 +104,23 @@ export default function getRecordInfo( postCount, postTypeBreakdown, lastIndexed
 	};
 }
 
+/**
+ * adds the appropriate labels the post type breakdown
+ *
+ * @param {Array} postTypeBreakdown - an array of the different post types with their counts
+ * @param {Array} postTypes - an array of the different post types labels matched with their slugs
+ * @returns {object} updated postTypeBreakdown containing the post type slug, label, and count
+ */
+export function addLabelsToPostTypeBreakdown( postTypeBreakdown, postTypes ) {
+	const postTypeBreakdownWithLabels = postTypeBreakdown.map( postType => {
+		const postTypeLabelItem = postTypes.find( label => label.slug === postType.slug );
```

Comment:
```suggestion
		const postTypeLabelItem = postType[ postType.slug ];
```
2. Locate the labels straight away from here, no need to search the array anymore